sdk/CMakeLists.txt

```diff
@@ -8,7 +8,9 @@ project("McmDPSdk" C)
 set(VERSION "0.1.0")
 set(DESCRIPTION "MCM DP SDK Library")
 
-set(CMAKE_C_STANDARD 11)
+#set(CMAKE_C_STANDARD 11)
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+set(CMAKE_C_STANDARD 99)
```

Comment:
This should not affect the client code as our SDK is build as a separate library so I assume that we can leave the standard of C as 11.

Reply:
Agree. Updated the PR.